env.example

```diff
@@ -1 +1,4 @@
 _STRIPE_SECRET_KEY=<REPLACE>
+DATABASE_USERNAME=<REPLACE>
```

Comment:
@SapeleD3 this is a very interesting project! 

Do we have a way of getting the Stripe Secret Key?

Reply:
@YizYah Yeah when you create a stripe acoount they give you api keys for sandbox and production